README.md

```diff
@@ -164,14 +165,17 @@ jobs:
         with:
           required-ros-distributions: ${{ matrix.ros_distribution }}
       - name: build and test
-        uses: ros-tooling/action-ros-ci@0.0.17
+        uses: ros-tooling/action-ros-ci@0.0.19
         with:
-          package-name: YOUR_PACKAGE_NAME
+          package-name: ${{ matrix.package }}
+          target-ros2-distro: ${{ matrix.ros_distribution }}
+          vcs-repo-file-url: ""
 
   test_docker: # On Linux, iterates on all ROS 1, and ROS 2 distributions.
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        package: [YOUR_PACKAGE, OR_PACKAGES]  # <-- Fill in the name of your package(s)
```

Comment:
This matrix approach isn't the most efficient way to build multiple packages as discussed - can we separate this bit from the version bumping? Simple version increases would be an easy approve :)

Reply:
sure! sorry :smile: